modules/@apostrophecms/notification/index.js

```diff
@@ -255,8 +259,8 @@ module.exports = {
               throw self.apos.error('invalid');
             }
             start = Date.now();
-            modifiedOnOrSince = req.query.modifiedOnOrSince && self.apos.launder.date(req.query.modifiedOnOrSince);
-            seenIds = req.query._id && self.apos.launder.ids(req.query._id) && req.query._id.split(',_id=');
+            modifiedOnOrSince = req.query.modifiedOnOrSince && new Date(req.query.modifiedOnOrSince);
+            seenIds = req.query.seenIds && self.apos.launder.ids(req.query.seenIds) && req.query._id.split(',_id=');
```

Comment:
I think you have an extra clause here so you are always going to get a split of _id rather than a laundering of seenIds.

Reply:
Done